lib/hooks/api/useContributorList.ts

```diff
@@ -43,16 +42,14 @@ export const useContributorsList = ({
     data: DbPRContributor[];
     meta: Meta;
   };
-  initialPage?: number;
+  page: number;
```

Comment:
no longer optional as we want to depend on it being passed in from the querystring.